tutorials/introductory/patchwork_axes.py

```diff
@@ -0,0 +1,195 @@
+"""
+==============
+Patchwork Axes
+==============
+
+This is inspired by the `patchwork
+<https://github.com/thomasp85/patchwork>`__ library for R.  While we
+do not implement the operator overloading style, we do provide a
+pythonic API for specifying (nested) Axes layouts.  If you need to do
+something that is not clearly expressed using this method, please see
+:ref:`sphx_glr_tutorials_intermediate_gridspec.py`.
+
+"""
+import matplotlib.pyplot as plt
+import numpy as np
+
+
+def label_axes(ax_dict):
+    for k, ax in ax_dict.items():
+        ax.set_title(k, loc="left")
+    return ax_dict
```

Comment:
Better not to return anything, as it clutters the output with the dictionary from being on the last line.